### PR TITLE
Feature spectrum repr

### PR DESCRIPTION
**Pull request by jccurtis**
With code created during #94, #107 this PR added functionality to printing `Spectrum` objects.

### CODE REVIEW

**Comment by markbandstra** [comment]
Needs testing.

**Comment by jccurtis** [comment]
Current format:

```
becquerel.Spectrum
    start_time:     None
    stop_time:      None
    realtime:       None
    livetime:       None
    is_calibrated:  True
    num_channels:   256
    gross_counts:   978+/-31
    gross_cps:      None
    filename:       None
```

anything else to add?

File: becquerel/core/spectrum.py [hunk 1]
```diff
@@ -203,6 +203,34 @@ def __init__(self, counts=None, cps=None, uncs=None, bin_edges_kev=None,
         else:
             self.infilename = None
 
+    def __repr__(self):
+        lines = ['becquerel.Spectrum']
```
Comment (markbandstra) [inline becquerel/core/spectrum.py]: Possibly include docstring

File: becquerel/core/spectrum.py [hunk 2]
```diff
@@ -203,6 +203,34 @@ def __init__(self, counts=None, cps=None, uncs=None, bin_edges_kev=None,
...
+        else:
+            ltups.append(('filename', None))
+        for lt in ltups:
+            lines.append('    {:15} {}'.format(
+                '{}:'.format(lt[0]),
+                lt[1]))
+        return '\n'.join(lines)
+
+    def __str__(self):
+        return self.__repr__()
```
Comment (markbandstra) [inline becquerel/core/spectrum.py]: @jccurtis Consider `__str__ = __repr__`

**Pull request merged**